lib/ITip/Broker.php

```diff
@@ -847,7 +847,7 @@ protected function parseEventInfo(VCalendar $calendar = null)
                     $organizer = $vevent->ORGANIZER->getNormalizedValue();
                     $organizerName = isset($vevent->ORGANIZER['CN']) ? $vevent->ORGANIZER['CN'] : null;
                 } else {
-                    if ($organizer !== $vevent->ORGANIZER->getNormalizedValue()) {
+                    if (strtolower($organizer) !== strtolower($vevent->ORGANIZER->getNormalizedValue())) {
```

Comment:
Logically doesnt make a difference, but I would use `strtoupper` as we use in the below lines

Reply:
Feel free to change to strtoupper, if you would like.